monkey/monkey_island/cc/next_ui/src/app/(protected)/_lib/useAuthRedirection.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+import { useRouter } from 'next/navigation';
+import { tokenStored } from '@/_lib/authentication';
+import { PATHS } from '@/constants/paths.constants';
+import { Events } from '@/constants/events.constants';
+import { useEffect } from 'react';
+
+export default function useAuthRedirection() {
+    const router = useRouter();
+
+    function checkToken() {
+        if (!tokenStored()) {
```

Comment:
rename to "isTokenStored" or "getStoredToken" (depends on the returned value)

Reply:
In the backend code we wouldn't add `is` because there's already a verb - `stored`. But I agree, in most cases prepending `is` makes the code more readable

Reply:
How about `tokenIsStored`?

`if token is stored` is more grammatically correct than `if is token stored`